.github/workflows/vineyard-operator.yaml

```diff
@@ -280,6 +280,10 @@ jobs:
           OUTPUT_DIR: ${{ github.workspace }}/${{ matrix.job }}-logs
           WITH_STOAT: true
 
+      - name: Setup tmate session
+        if: ${{ failure() }}
```

Comment:
Use `if: false`, and enable it when you want to debug.

Reply:
Done.